sample-app/build.gradle

```diff
@@ -1,7 +1,8 @@
 buildscript {
     repositories { jcenter() }
     dependencies { 
-        classpath 'com.android.tools.build:gradle:0.12.+' // 0.12.x is for Gradle 1.12 projects
+        classpath 'com.android.tools.build:gradle:1.+' // Android gradle 1.+ is for Gradle 2.+ projects
```

Comment:
Remove the comment its redundant. Also please specify the while version. Wildcard dependencies are an atrocity